integration-tests/src/test/resources/config-test-child/invalid-content.properties

```diff
@@ -0,0 +1,17 @@
+# content.properties - Metadata for the content package and its dependencies
+
+# The name of this content package
+name=openmrs-content-hiv
+
+# The version of this content package (must follow SemVer)
+version=latest
+
+# Dependencies (specified in a format similar to distro.properties)
+
+# OMOD modules with their SemVer range
+omod.fhir2=>=1.0.0-SNAPSHOT
+omod.reporting=^2.0.0
+
+# SPA modules with their SemVer range
+spa.frontendModules.@openmrs/esm-login-app=^3.0.0
+spa.frontendModules.@openmrs/esm-patient-chart=~1.0.0
```

Comment:
```suggestion
spa.frontendModules.@openmrs/esm-patient-chart=~1.0.0

```

Reply:
Done! I thought you won't like that empty line, looks like its needed